jbpm-runtime-manager/src/test/java/org/jbpm/runtime/manager/concurrent/ConcurrentOperationsTest.java

```diff
@@ -139,6 +140,7 @@ public void testExecuteProcessWithAsyncHandler() throws Exception {
         manager.close();
     }
     
+    @Ignore("unstable")
```

Comment:
So you haven't found out why it behaves so poorly?

Reply:
Unfortunately I didn't...